packages/react/src/Pagination/Pagination.tsx

```diff
@@ -98,7 +97,7 @@ type UsePaginationPagesParameters = {
   hrefBuilder: (n: number) => string
   pageAttributesBuilder?: (n: number) => {[key: string]: string}
   marginPageCount: number
-  showPages?: PaginationProps['showPages']
+  showPages: boolean
```

Comment:
On line 37 the same value can also be undefined, should that be the case here too?

Reply:
No, there's no reason for this hook to accept `showPages` as being `undefined` as L53 sets a default value of `showPages` to be `true`.

So `showPages` can be `undefined` when passed to the `Pagination` component, however the `Pagination` will then make sure it's definitely a boolean before passing it to the hook.